css/amp.css

```diff
@@ -803,6 +803,18 @@ amp-accordion > section[expanded] > :last-child {
   width: 100%;
 }
 
+amp-story[standalone] {
+  background-color: #fff !important;
```

Comment:
What if the author wants to override this?

Reply:
This is rarely ever shown.  This is the background of the story itself, but not the pages, and all of our experiences show the background of the page (which the publisher can override).  This should only show up while the story is loading.

Reply:
Right, I missed that. Thanks.